cmake/external_libs/gazebo.cmake

```diff
@@ -1,10 +1,7 @@
 # If Gazebo is added as a dependency multiple times (e.g. from
```

Comment:
I guess this comment is a little out of date?

Reply:
True!